kaspresso/src/main/kotlin/com/kaspersky/kaspresso/systemsafety/SystemDialogSafetyProviderImpl.kt

```diff
@@ -114,6 +115,7 @@ class SystemDialogSafetyProviderImpl(
 
             if (systemDialogsSafetyParams.shouldIgnoreKeyboard) {
                 val isKeyboardVisible = isVisible(By.pkg(Pattern.compile("\\S*google.android.inputmethod\\S*")).clazz(FrameLayout::class.java))
+                logger.i("Found keyboard")
```

Comment:
is this comment still valid here?

Reply:
Yeah, it could go into the void. Removed it with the last commit. I'll squash on merge